hooks/post_gen_project.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+def parse_dvc_settings():
+    if "{{ cookiecutter.data_version_control }}" == "no":
```

Comment:
Make sure to change this to `if "{{ cookiecutter.use_data_version_control }}" == "no":`